worldedit-core/src/main/java/com/fastasyncworldedit/core/extension/factory/parser/pattern/BufferedPattern2DParser.java

```diff
@@ -33,14 +35,21 @@ protected Stream<String> getSuggestions(String argumentInput, int index) {
 
     @Override
     public Pattern parseFromInput(@Nonnull String[] input, ParserContext context) throws InputParseException {
+        Region selection = null;
```

Comment:
We can probably outsource this one into an agnostic protected method, considering it's basically the same code in BufferedPatternParser.